glide.yaml

```diff
@@ -45,7 +45,8 @@ import:
 - package: github.com/openshift/client-go
   version: master
 - package: github.com/openshift/library-go
-  version: master
+  repo:    https://github.com/soltysh/library-go.git
+  version: rebase-1.12.2
```

Comment:
why did this change?

Reply:
That's part of my drop commit, that will be removed once we merge https://github.com/openshift/library-go/pull/78

Reply:
This is already dropped.